src/utils/CommandExecutor.js

```diff
@@ -0,0 +1,170 @@
+import fs from 'fs';
+import path from 'path';
+import request from 'request';
+import csvParse from 'csv-parse';
+import through2 from 'through2';
+import { promisify } from 'util';
+
+const readDir = promisify(fs.readdir);
+
+export default class CommandExecutor {
+  constructor() {
+    this.csvFolderPath = `${__dirname}/../../data/csv/`;
+    // acceess to file in tasks (https://epa.ms/nodejs18-hw3-css) forbidden;
+    this.cssUrl = 'https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.css';
+    this.cssBundleName = 'bundle.css';
+  }
+
+  reverse() {
+    console.log('Type something to reverse, please:');
+    process.stdin
+      .pipe(this.reverseInput())
+      .pipe(process.stdout);
+  }
+
+  transform() {
+    console.log('Type something to transforme, please:');
+    process.stdin
+      .pipe(this.toUpperCase())
+      .pipe(process.stdout);
+  }
+
+  outputFile(csvName) {
+    if (!csvName) {
+      console.error('File is not specified');
+      return;
+    }
+
+    const csvPath = this.getFullCsvPath(csvName);
+
+    fs.createReadStream(csvPath)
+      .pipe(csvParse())
+      .pipe(this.csvToString())
+      .pipe(process.stdout);
+  }
+
+  convertFromFile(csvName) {
+    if (!csvName) {
+      console.error('File is not specified');
+      return;
+    }
+
+    const csvPath = this.getFullCsvPath(csvName);
+
+    fs.createReadStream(csvPath)
+      .pipe(csvParse())
+      .pipe(this.csvToObject())
+      .pipe(this.objectToString())
+      .pipe(process.stdout);
+  }
+
+  convertToFile(csvName) {
+    if (!csvName) {
+      console.error('File is not specified');
+      return;
+    }
+    const csvPath = this.getFullCsvPath(csvName);
+    const writeStream = fs.createWriteStream(csvPath.replace(/csv/g, 'json'));
+
+    fs.createReadStream(csvPath)
+      .pipe(csvParse())
+      .pipe(this.csvToObject())
+      .pipe(this.objectToString())
+      .pipe(writeStream);
+  }
+
+  cssBundler(directory) {
+    if (!directory) {
+      console.error('Path is not specified');
+      return;
+    }
+
+    const outputFile = path.resolve(`${directory}/${this.cssBundleName}`);
+
+    readDir(directory)
+      .then(files => files.filter(fileName => fileName.indexOf('.css') !== -1 && fileName !== this.cssBundleName))
+      .then(files => {
+        files.forEach(file => {
+          const writeStream = fs.createWriteStream(outputFile, { flags: 'a' });
```

Comment:
If there will be big files with more than one chunk inside, you'll have some mess in the end. Because you read files simultaneously and pipe them to the same `bundle.css`.